src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md

```diff
@@ -7,18 +7,75 @@ A *cache type* enables you to specify what is cached and enables merchants to cl
 
 The tag *scope* provides a mechanism for a cache type.
 
-To create a new cache type:
+## Adding a custom cache type {#m2devgde-adding-custom-cache-type}
+
+This topic discusses how to create a custom cache type.
+
+### Cache type configuration {#m2devgde-cache-type-configuration}
+
+Declare a new cache type in the `etc/cache.xml` file with the following attributes:
+
+| Attribute | Required? | Description |
+| --- | --- | --- |
+| `name` | Yes | A unique cache type ID |
+| `translate` | No | Parameters that would be translated during to render "Cache Management" page |
+| `instance` | Yes | The cache type model |
```

Comment:
```suggestion
| `instance` | Yes | The cache type model class |
```

Reply:
Hello @rogyar. Thank you for your remark. I fixed it. Please check.